jha/src/components/enrolment/Child.vue

```diff
@@ -1480,6 +1480,7 @@ export default {
         this.citizenshipSupportDocuments = [];
         this.isNameChanged = null;
         this.livedInBCSinceBirth = null;
+        this.previousHealthNumber = null;
```

Comment:
Looks good to me, just asking for my own knowledge. Why is it just the previous health numbers that vuelidate doesn't need to call `$reset` for? Is that from a previous page

Reply:
`previousHealthNumber` is always optional and doesn't have any validation on it, so there's no need to reset the vuelidate object